### ISSUE TITLE
Improve map error view

### ISSUE THREAD

**Issue by weeman1337** [vector-im/element-web#22588]
#### What would you like to do?

As an user I don't want to be scared away by a map error.

Here is what it currently looks like:

![image](https://user-images.githubusercontent.com/6216686/173796598-9300a111-0fef-456d-af80-a5cf82c69a09.png)

#### Why would you like to do it?

Improve the UI. Let it look nicer.

#### How would you like to achieve it?

- Use a map placeholder
- Hide info about „home server“ and „tile server“ because that does not provide info for the common user
- Instead use some text like „Unfortunately the map couldn't be loaded :information_source: “ More details can be put behind :information_source: as a tooltip or popover

We already have a placeholder for expired live shares:

![image](https://user-images.githubusercontent.com/6216686/173797980-c21960f3-02d7-4112-b0d8-7bc99d631299.png)

**Comment by kerryarchibald** [comment]
Discussed with Gaelle, static location shares should use the same 'map server unavailable error' UI as live location shares
Reusable component here: https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/location/MapError.tsx

<img width="521" alt="Screenshot 2022-07-20 at 11 37 25" src="https://user-images.githubusercontent.com/3055605/179950433-c3f9f629-8fe4-4d8a-9227-81c484a1c0f5.png">

https://www.figma.com/file/uthEK6xuo9hSQRdWvPAre5/Location-Sharing?node-id=2302%3A64561

**Comment by weeman1337** [comment]
„home server“ is alien speech :space_invader: 

What about changing it to

„Unable to load map :information_source:“

Where :information_source: can have a tooltip with the content „This home server is not configured to display maps. Contact your friendly admin.“.

**Comment by gaelledel** [comment]
Yes to add "Contact your admin for more help" After "This home server is not configured to display maps."

**Comment by t3chguy** [comment]
I think homeserver should be one word not two
